draft-ietf-quic-multipath.md

```diff
@@ -240,12 +240,11 @@ ACK frames when used with the multipath extension acknowledge packets for the pa
 As multipath support is unknown during the handshake, acknowledgments of
 Initial and Handshake packets are sent using ACK frames.
 
-After the handshake concluded and both endpoints support the multipath extension,
+After the handshake concluded with support for the multipath extension,
 endpoints SHOULD use PATH_ACK frames instead of ACK frames,
-including for acknowledging so far unacknowledged 0-RTT packets using Path ID 0.
-Endpoints MUST process ACK frames that acknowledge 0-RTT packets or 1-RTT packets,
-even when this extension is used. For example,
-ACK frames might be preferred by the sender as long as only the initial path
+also so far unacknowledged 0-RTT packets using Path ID 0.
```

Comment:
The initial writing is better. "also" is a bit ambiguous, and "also so far" does not really convey the meaning. I suggest reverting to "including for acknowledging so far unacknowledged 0-RTT packets using Path ID 0."

Reply:
Agree with Christian here. The "including" version is better.